<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuPage</class>
 <widget class="QWizardPage" name="MenuPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Select option:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioCreate">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>Create new wallet</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioOpen">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>Open wallet file</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioSeed">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>Restore wallet from seed</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioViewOnly">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>Import from keys</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>by dsc &amp; tobtoht</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>banner: themonera.art</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
